ros_robot_controller/PWMServoState[] state

float64 duration
